subsys/bluetooth/shell/ticker.c

```diff
@@ -0,0 +1,124 @@
+/** @file
+ * @brief Bluetooth Controller Ticker functions
+ *
+ */
+
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <shell/shell.h>
+#include <misc/printk.h>
+
+#include "../controller/util/mayfly.h"
+#include "../controller/ticker/ticker.h"
+
+#define TICKER_SHELL_MODULE "ticker"
+
+#if defined(CONFIG_BLUETOOTH_MAX_CONN)
+#define TICKERS_MAX (CONFIG_BLUETOOTH_MAX_CONN + 2)
+#else
+#define TICKERS_MAX 2
+#endif
+
+static void ticker_op_done(u32_t err, void *context)
+{
+	*((u32_t volatile *)context) = err;
+}
+
+int cmd_ticker_info(int argc, char *argv[])
```

Comment:
This implementation is specific to displaying, additional information about occupied timespace is not part of the used ticker_next_slot_get and will be a new interface in ticker. Each user (shell, controller, flash and others) would like to implementation enumeration and use of information returned in their own ways, hence moving this as ticker API may not be desired.